src/dmd/dsymbolsem.d

```diff
@@ -3953,6 +3953,8 @@ private extern(C++) final class DsymbolSemanticVisitor : Visitor
                     {
                         OutBuffer buf1;
 
+                        if (fd.toChars() == funcdecl.toChars())
+                            hgs.fullQual = true;
```

Comment:
Those both return pointers... So we're just comparing the pointers ?

Reply:
Hmm.. I was under the impression that was changed to return a slice.

Reply:
You wouldn't be able to call it in `printf` and `error` if that was the case ;)